harmoni_core/harmoni_pattern/test/test_sequential_speak.py

```diff
@@ -22,20 +22,24 @@
 
 
 class TestSequential(unittest.TestCase):
-    """This test of the sequential pattern player uses a simple mic test script.
+    """This test of the sequential pattern player uses a speak test script.
 
-    If the mic is launched and starts successfully (as verified by the subscriber)
-    then the test is passed.
+    This is a sequence of:
+    - bot
+    - tts
+    - speak and lip-sync
+    If all the requests succeed, the test passes.
```

Comment:
I suppose it's okay for now, but long term I would want to make sure it runs correctly for any possible sequential sequence, not just testing some specific sequence.